backend/curriculum_tracking/serializers.py

```diff
@@ -184,15 +192,37 @@ class Meta:
             "oldest_open_pr_updated_time",
             "repo_url",
             "users_that_reviewed_since_last_review_request",
+            "total_nyc_and_rf_reviews"  #**
         ]
 
     users_that_reviewed_since_last_review_request = serializers.SerializerMethodField(
         "get_users_that_reviewed_since_last_review_request"
     )
+    total_nyc_and_rf_reviews = serializers.SerializerMethodField("get_total_nyc_and_rf_reviews") #**
 
     def get_users_that_reviewed_since_last_review_request(self, instance):
         return instance.get_users_that_reviewed_since_last_review_request()
 
+    #**
+    def get_total_nyc_and_rf_reviews(self, card):
+         negative_reviews = models.RecruitProjectReview.objects.filter(status__in=['NYC', 'R']).filter(recruit_project__agile_card=card).count()
+         return negative_reviews
+
```

Comment:
The way you use the serializer and how you set up your function "get_total_nyc_and_rf_reviews" is a bit strange.  For instance, if you look at the function above "get_users_that_reviewed_since_last_review_request()", you will notice that the function returns details of the instance of the class, ergo, an instance/object of the AgileCard class.

In your function you first gather info about RecruitProjectReview objects which match your criteria.  This is not wrong since you are getting the data you want, I am just now sure if it is the most efficient way of going about it.  I think that if you are going to do it this way you might as well not use a serializer at all and simply have a view and inside the view have a small function which takes care of business (much like the example I showed you where I also did not use any serialization at all).  However, wait for Sheena to confirm this first.